### DISCUSSION TITLE
961c151d2e87f2686a955a9be24d316f1362bf21

### DISCUSSION THREAD

**Discussion by hexacorn** [nlohmann/json#4123] [General]
May I ask where 961c151d2e87f2686a955a9be24d316f1362bf21 comes from in this error message?  is it a SHA1 of something? ta!
https://github.com/nlohmann/json/blob/5d2754306d67d1e654a1a34e1d2e74439a9d53b3/include/nlohmann/json.hpp#L519

**Comment by nlohmann** [comment]
It's

```shell
echo "JSON for Modern C++" | sha1sum
```

**Reply by hexacorn** [reply]
Thanks a lot, I posted this short article about your answer here: [https://www.hexacorn.com/blog/2023/09/03/the-secret-of-961c151d2e87f2686a955a9be24d316f1362bf21/](https://www.hexacorn.com/blog/2023/09/03/the-secret-of-961c151d2e87f2686a955a9be24d316f1362bf21/)
